test/verify/check-kdump

```diff
@@ -94,8 +94,12 @@ class KdumpHelpers(testlib.MachineCase):
 @testlib.skipImage("kexec-tools not installed", "debian-*", "ubuntu-*", "arch")
 @testlib.timeout(900)
 class TestKdump(KdumpHelpers):
+    provision = {
+        "0": {"memory_mb": 1024},
+    }
```

Comment:
Why not keep the default (1152) for these? Lowering it might not go well

Reply:
I realize the later commit changes the number, but it's still odd/wrong in the series.